masonry/README.md

```diff
@@ -16,6 +16,11 @@
 
 [tracing_tracy]: https://crates.io/crates/tracing-tracy
```

Comment:
This line is the equivalent of the `libm` line in: https://github.com/linebender/color/blob/9775885b74e37f2de2a1713f80c607ba84e474c1/color/README.md#L21-L23

Therefore, it should be below the new comment and the links block should have the same comment as in color.